Makefile

```diff
@@ -273,11 +296,14 @@ release: ## Create a new release from the current beta and push to github
 .PHONY:build-image
 build-image: ## Build a docker image as specified in the Dockerfile
 	$(DOCKER) build . -t $(REPO):$(BUILD_TAG) \
-		--pull \
+		$(PULL_OPTS) \
```

Comment:
Curious, why the change to pull newer image?

Reply:
This just makes it use the PULL_OPTS option that was already in the makefile, it shouldn't change the behaviour except that it means we can override it by setitng PULL_OPTS=0

Reply:
makes sense